src/scripts/content/freshdesk.js

```diff
@@ -31,8 +31,13 @@ togglbutton.render('.page-actions__left:not(.toggl)', { observe: true }, elem =>
 
   const descriptionSelector = () => {
     const ticketNumber = $('.breadcrumb__item.active').textContent.trim();
-    const subject = $('.ticket-subject-heading').textContent.trim();
-    return `${ticketNumber} ${subject}`;
+    // Remove other buttons/controls from the ticket subject
+    const subjectElement = $('.ticket-subject-heading').cloneNode(true);
+    for (let i = 0; i < subjectElement.children.length; i++) {
```

Comment:
Not something major but I'd prefer a for-of or for-each loop

```suggestion
    for (let child of subjectElement.children) {
        subjectElement.removeChild(child);
```

or 

```suggestion
    [...subjectElement.children].forEach(child => subjectElement.removeChild(child));
```